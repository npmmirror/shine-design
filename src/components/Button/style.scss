/**
 * @File 
 * @Author qulongjun@shine.design
 * @Date 2021/5/8 20:46:05
 */
 button {
   display: flex;
 }