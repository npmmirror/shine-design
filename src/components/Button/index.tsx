/**
 * @File Demo Button
 * @Author qulongjun@shine.design
 * @Date 2021/5/8 14:25:23
 */
import React from "react";
import './style.scss';

export default () => {

    return (
        <div>
            This is a Button Test
        </div>
    );
};