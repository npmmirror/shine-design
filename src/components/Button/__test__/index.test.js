/**
 * @File Demo Test
 * @Author qulongjun@shine.design
 * @Date 2021/5/8 14:26:08
 */
import React from 'react'
import Button from '../index';
import Enzyme from '../../../utils/jest.setup';
import toJson from 'enzyme-to-json';

const { render } = Enzyme;

describe('Button测试', () => {
    it('Example组件中按钮的名字为text的值', () => {
        const button = render(<Button>button</Button>);
        expect(toJson(button)).toMatchSnapshot();
    })
});