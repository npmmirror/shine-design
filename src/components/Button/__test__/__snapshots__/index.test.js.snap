// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button测试 Example组件中按钮的名字为text的值 1`] = `
<div>
  This is a Button Test
</div>
`;
